src/test/java/io/jenkins/plugins/analysis/core/model/ByIdResultSelectorTest.java

```diff
@@ -0,0 +1,64 @@
+package io.jenkins.plugins.analysis.core.model;
```

Comment:
Overall formatting: reformat with IntelliJ defaults. Should hopefully use the settings of [my coding style](https://github.com/uhafner/codingstyle)